### DISCUSSION TITLE
Results Section in Data Quality Dashboard Has Nothing To Display

### DISCUSSION THREAD

**Discussion by ranatech08** [OHDSI/DataQualityDashboard#345] [General]
Hi,

I have managed to run the Data Quality Dashboard on the Synthea dataset, however the **“Results” tab is showing blank**. Please note that the “**Overview**” & “**Metadata**” fields are being populated.

**Results Tab with no Data:**

![No Results in Results Tab](https://user-images.githubusercontent.com/34309448/183096505-0c8abd56-f2aa-4fa8-adde-08f22e7cdd44.png)

**Overview & Metadata with Data:**
![Overview image](https://user-images.githubusercontent.com/34309448/183096813-7275b45e-39d2-4cef-b463-c31eebdea561.jpg)
![Metadata image](https://user-images.githubusercontent.com/34309448/183097014-a6c1dbce-014e-432b-ab27-2967b65f2fc3.png)

I ran the `viewDqDashboard `command in R that created the `results.json` file in the `home `directory. Below is image of home directory:
![results json file](https://user-images.githubusercontent.com/34309448/183097393-92242b40-76ba-4f1e-9212-6c75010dc1c1.png)

I have even checked the path of json where the `results.json` is generated by printing it out. This path is correctly pointing to requisite path. Image below:
![json file path](https://user-images.githubusercontent.com/34309448/183097581-34de95a3-54b3-4942-b546-6a0afc35d0de.png)

Would be great if someone can please direct me on what needs to be done.

Thanks in advance.

Best,
Rana

**Comment by katy-sadowski** [comment]
Apologies for the delay - I am just now seeing some of these old Discussions posts. @ranatech08 if you are still having issues with the viewer please open a new issue [here](https://github.com/OHDSI/DataQualityDashboard/issues). Thank you!
